### ISSUE TITLE
EOR Screen Filtering

### ISSUE THREAD

**Issue by PursuitInAshes** [RMC-14/RMC-14#10677]
## Description
EOR Screen should have the ability to filter by job title or by IC character name to make it easier to parse. This isn't an issue on other servers because they both have significantly fewer players and less consistent ghost roles to "clog" the UI.
